lib/app/layouts/conversation_view/widgets/text_field/text_field_suffix.dart

```diff
@@ -73,7 +75,7 @@ class _TextFieldSuffixState extends OptimizedState<TextFieldSuffix> {
                       ? null
                       : !isChatCreator && !showRecording
                       ? context.theme.colorScheme.outline
-                      : context.theme.colorScheme.primary,
+                      : CupertinoDynamicColor.resolve(iOSStopIconBackground, context),
```

Comment:
Make this `context.theme.colorScheme.primary.withOpacity(0.5)` (can modify opacity so it matches iOS opacity best)